src/main/java/com/SocialMedia/SocialMedia/models/dto/UserDTO.java

```diff
@@ -1,4 +1,22 @@
 package com.SocialMedia.SocialMedia.models.dto;
 
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDate;
+@Getter
+@Setter
+
 public class UserDTO {
+
+    public String firstName;
```

Comment:
- @cristi6891 maybe we need use fundamental OOP concepts Encapsulation (can't be accessible outside the class) ?  
More info:
 https://stackoverflow.com/questions/48820066/why-do-we-declare-private-variables-in-java

Reply:
changed from public to private

Reply:
ok, good job